### PR TITLE
Finished Homepage

### PR DESCRIPTION
**Pull request by GarvG**
Completed My Home Page for Ecommerce-using React The Homepage Include
1) Navbar
2) Cover Page
3) Categories to Choose from

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          <div className="left">
+                <img className="logo"src={logo}></img>
+          </div>
+          <div className="center">
+
+              <input placeholder='Search'></input>
+          </div>
+          <div className="right">
+              <ul className="right-navbar">
+                  <li className="Icons">Home </li>
```
Comment (rutu14) [inline src/components/Navbar.jsx:17]: Suggestion: Since this are repeating, can create a component to reuse it.

**Review by rutu14 — APPROVED** [review]
Suggestion:
Create a index file in components and image folder and get all imports of the folder there and export them. This way you can use them from one file only.

Overall, the structure is good to merge.

File: src/components/FeaturedCategories.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import "../styles/FeaturedCategories.css"
+const FeaturedCategories = () => {
+  return (
+    <div className="Categories">
+        <h2 className='content'> Categories to Choose From...</h2>
+        <div className="small-container">
+            <div className="row">
+                <div className="col-3">
+                    <p className='img-tag'>Air Max Low</p>
+                <img src={low}></img>
```
Comment (santhoshpemmaka) [inline src/components/FeaturedCategories.jsx:14]: suggestion: if possible add alt tag in the inside image.

**Review by santhoshpemmaka — APPROVED** [review]

**Comment by santhoshpemmaka** [comment]
The code looks clean and good 💯 . Good to merge 👍

File: src/components/FeaturedCategories.jsx (hunk 2 above)
Comment (GarvG) [reply src/components/FeaturedCategories.jsx:14]: will take care from next time

File: src/components/Navbar.jsx (hunk 1 above)
Comment (GarvG) [reply src/components/Navbar.jsx:17]: Sure will take care from next time

**Pull request merged**